lib/mshoplib/src/MShop/Plugin/Provider/Order/PropertyAdd.php

```diff
@@ -181,15 +194,15 @@ protected function addAttributes( \Aimeos\MShop\Order\Item\Base\Product\Iface $o
 	/**
 	 * Returns the product items for the given product IDs limited by the map of properties
 	 *
-	 * @param string[] $productIds List of product IDs
+	 * @param string[] $productCodes List of product codes
 	 * @return \Aimeos\Map List of items implementing \Aimeos\MShop\Product\Item\Iface with IDs as keys
 	 */
-	protected function getProductItems( array $productIds ) : \Aimeos\Map
+	protected function getProductItems( array $productCodes ) : \Aimeos\Map
 	{
 		$manager = \Aimeos\MShop::create( $this->getContext(), 'product' );
 		$search = $manager->filter( true );
 		$expr = [
-			$search->compare( '==', 'product.id', array_unique( $productIds ) ),
+			$search->compare( '==', 'product.code', array_unique( $productCodes ) ),
 			$search->getConditions(),
 		];
 		$search->setConditions( $search->and( $expr ) );
```

Comment:
You don't use both, IDs and codes

Reply:
My bad, I never copied my changes over, im copying my edits from editor to the browser.

Im trying to find more info on how I can run the test suite using phing, I just setup the environment so it has some data to work against.